index.html

```diff
@@ -0,0 +1,199 @@
+          target="_blank">@maryna_r</a></li>
+    </ul>
+    <div class="summary">
+      <p>Beginning energetic Front-End Developer with strong desire to get ton of enjoyable daily web development
+        challenges at EPAM.
+        My strong work ethic and prior work experience will ensure that I make meaningful contributions to your
+        company’s goals. As an organized and hard-working individual, I will add significant value to the internship
+        role at EPAM.</p>
+    </div>
+  </header>
+
+  <main>
+    <section class="skills">
+      <h3>Skills</h3>
+      <div class="container">
+        <div class="card">
+          <div class="top-card">
+            <p>Hard Skills</p>
+          </div>
+          <div class="bottom-card">
+            <ul class="card-list">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+              <li>Git</li>
+              <li>Adobe Photoshop</li>
+            </ul>
+          </div>
+        </div>
+        <div class="card">
+          <div class="top-card">
+            <p>Soft Skills:</p>
+          </div>
+          <div class="bottom-card">
+            <ul class="card-list">
+              <li>self-management</li>
+              <li>ability to listen</li>
+              <li>ability to ask questions</li>
+              <li>business writing skills</li>
+              <li>ability to manage the stress</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="code-examples">
+      <h3>Code examples</h3>
+      <div class="container">
+        <div class="card">
+          <div class="top-card">
+            <p>Example of Code</p>
+          </div>
+          <div class="bottom-card">
+            <pre class="prettyprint">
+                var currentTime = new Date(),
+                  hours = currentTime.getHours(),
+                  minutes = currentTime.getMinutes();
+              
+              // Add a leading zero if needed
+              if (minutes < 10) {
+                minutes  = "0" + minutes; 
+              }
+              
+              // Join the parts
+              time = hours + ":" + minutes;
+                  
+              // Output the result to the above HTML element
+              document.getElementById("time").innerHTML = time;
+            </pre><br>
+            <p><a href="https://github.com/maryna-js" class="code-link" target="_blank">Other code examples by
+                reference</a></p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="experience">
+      <h3>Experience</h3>
+      <div class="container">
+        <div class="card">
+          <div class="top-card">
+            <p>Sumka.by</p>
+          </div>
+          <div class="bottom-card">
+            <p>Online web store <a href="https://sumka.by/" target="_blank">sumka.by</a><br>
```

Comment:
tag br is not good. Maybe it would be good if you use another tag and line break is configured using css